chrome_extension/background.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file background.js
+ * @description Creates a new background script that listens for messages from the content script
+ * and sends them to a WebSocket server.
+*/
+
+let socket;
+
+/*
+ * Function to connect to the WebSocket server.
+*/
+function connectWebSocket() {
+  socket = new WebSocket("ws://localhost:8765");
```

Comment:
What do you think about defining the port as a named constant at the top of the file?

Ideally we read it from `config.py`, or it gets passed in somehow. Maybe we can leave it as a TODO.

Reply:
Done here: dc5c69953ceca8d3a0b0598b1226262be472ab0a